live-server.js

```diff
@@ -150,7 +148,9 @@ for (var i = process.argv.length - 1; i >= 2; --i) {
 }
 
 // Patch paths
-var dir = opts.root = process.argv[2] || "";
+opts.root = opts.root || process.argv[2] || process.cwd();
```

Comment:
perhaps I could flip `opts.root` and `process.argv[2]`...  up to you.

Reply:
I think options given on command line should always take precedence.

Reply:
Sure thing - i will swap them. I didnt want to change what was there already but i raised it since it also made sense to me the cli option should take higher precedence.